src/components/Editor/EditorPanel/EditorPanel.jsx

```diff
@@ -164,7 +164,7 @@ const EditorPanel = ({ extension = "html", fileName = "index" }) => {
   }, [file, cascadeUpdate, editorViewRef]);
 
   return (
-    <>
+    <div className={isDarkMode ? "dark" : "light"}>
```

Comment:
I think we already have class names more globally that could be used for some of this? Unless we got rid of them 🤔